lib/lotus/repository.rb

```diff
@@ -674,19 +674,15 @@ def _persisted?(entity)
 
       # Add time create an entity
       def _update_created_at(entity)
-        if entity.class.attributes.include?(:created_at)
+        if entity.respond_to?(:created_at=)
           entity.created_at ||= Time.now.utc
         end
       end
 
       # Add time update an entity
       def _update_updated_at(entity)
-        if entity.class.attributes.include?(:updated_at)
-          if entity.updated_at
-            entity.updated_at = Time.now.utc
-          else
-            entity.updated_at = entity.created_at
-          end
+        if entity.respond_to?(:updated_at=)
+          entity.updated_at ? entity.updated_at = Time.now.utc : entity.updated_at = entity.created_at
```

Comment:
I think we will have 3 cases:
* Has created_at (Done)
* Has updated_at (Not yet done)
* Have created_at & updated_at (Done & will update condition ensure that entity respond to to `updated_at` and `created_at`)